modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -323,7 +323,14 @@ export default class LargeVideoManager {
      * Update container size.
      */
     updateContainerSize() {
-        this.width = UIUtil.getAvailableVideoWidth();
+        let widthToUse = UIUtil.getAvailableVideoWidth();
+        const { isOpen } = APP.store.getState()['features/chat'];
+
+        if (isOpen) {
+            widthToUse -= 375;
```

Comment:
can you put a comment about where this width comes from?

Reply:
Done